modules/network-observability-lokistack-create.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-lokistack-create_{context}"]
+
+= Create a LokiStack custom resource 
+
+.Procedure
+
+. Navigate to *Operators* -> *Installed Operators*. In the Details, under *Provided APIs*, select `LokiStack` and click *Create LokiStack*. 
+. Ensure the following fields are specified in either *Form View* or *YAML view*:
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: loki
+    namespace: netobserv
+  spec:
+    size: 1x.small
+    storage:
+      schemas:
+      - version: v12
+        effectiveDate: '2022-06-01'
+      secret:
+        name: loki-s3
+        type: s3
+    storageClassName: gp3
```

Comment:
We should annotate the storageClassName, with a note saying roughly to use a storage class name for "ReadWriteOnce" access mode (this comes from feedback that we got from some users)

(Note, on my dev cluster I don't have the `gp3` class name by default, I need to use `gp2-csi`)

Reply:
Added